task_list.gemspec

```diff
@@ -16,15 +16,18 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "rack", "~> 1.0"
-  gem.add_dependency "activesupport", "~> 4.0"
+  gem.required_ruby_version = ">= 2.1.0"
+
+  gem.add_dependency "activesupport", "~> 4.0" if RUBY_VERSION < '2.2.2'
+  gem.add_dependency "nokogiri", "~> 1.6.0" if RUBY_VERSION < '2.1.0'
```

Comment:
@connorshea I updated the gemspec to only pin the dependencies when the ruby version is too old.

```
ruby 1.9.3p551 (2014-11-13 revision 48407) [x86_64-linux]
Installing rack 1.6.5
Installing nokogiri 1.6.8.1
Installing activesupport 4.2.8
```

```
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
Using rack 2.0.1
Using nokogiri 1.7.1
Using activesupport 5.0.2
```